demos/social_distance_demo/cpp/src/geodist.cpp

```diff
@@ -0,0 +1,307 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <cmath>
+#include <tuple>
+#include <vector>
+#include "geodist.hpp"
+
+typedef std::pair<cv::Point2d, cv::Point2d> Line2d;
+
+std::pair<double, double> getLineComponent(const cv::Point2d &p1, const cv::Point2d &p2) {
+    double run = p2.x - p1.x;
+    double rise = p2.y - p1.y;
+    double a = 0;
+
+    if (std::fabs(run) < 0.0000001) {
+        double sign = (run > 0) - (run < 0);
```

Comment:
```suggestion
        double sign = run < 0 ? -1.0 : 1.0;
```
If run == 0 then sign == 0, which leads to division by zero.

Reply:
Ok, changed.